Core/Src/cockpit/indicators.c

```diff
@@ -0,0 +1,62 @@
+#include "indicators.h"
+
+#ifdef UNIT_TEST
+    struct IndicatorsHandler indicators_global_handler; // Visible to tests
+#else
+    static struct IndicatorsHandler indicators_global_handler; // Private in production
```

Comment:
```suggestion
    EAGLETRT_STATIC struct IndicatorsHandler indicators_global_handler; // Private in production
```

You shall add eagletrt's library (https://github.com/eagletrt/libeagletrt-sw) as a platformio dependency.
This should be updated in other similar instances as well.